packages/router/src/bindings/prices/index.ts

```diff
@@ -22,10 +22,10 @@ export const bindPrices = async () => {
         const cachingAssetIdOnMainnet = await getMainnetEquivalent(asset.assetId, asset.chainId);
         const cachingTokenChainId = cachingAssetIdOnMainnet ? 1 : asset.chainId;
         const cachingTokenAssetId = cachingAssetIdOnMainnet ? cachingAssetIdOnMainnet : asset.assetId;
-        if (!chainAssetMap.get(cachingTokenChainId)) {
+        if (!chainAssetMap.has(cachingTokenChainId)) {
```

Comment:
why was `get` not working before? wouldn't it have returned undefined if didn't exist?

no possibility of it returning 0 so idk if i see the problem lol, lmk

Reply:
just curious. `has` is better ofc

Reply:
oh bc it's an array